src/components/NewClusterModal.js

```diff
@@ -405,6 +470,22 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
           h(ButtonPrimary, { onClick: () => this.createCluster() }, ['REPLACE'])
         ])
       ])],
+      ['update', () => h(Fragment, [
+        this.isStopRequired()
+          ? p(['Changing the machine type (increasing or decreasing the # of CPUs or Mem) results in an update that requires a ',
+            b(['restart']),
+            ' of your runtime. This may take a few minutes.  Would you like to proceed? ',
+            b(['(You will not lose any files.)'])])
+          : p(['Increasing the disk size or changing the number of workers (when the number of workers is >2) results in a real-time update to your runtime. ',
+            'During this update, you can continue to work']),
```

Comment:
Nit: I know Isaac mentioned something about the formatting of this `?:` but I'm not 100% sure what the more conventional way is. It might be something like:
```
        this.isStopRequired() ? p([
          'Changing the machine type (increasing or decreasing the # of CPUs or Mem) results in an update that requires a ',
          b(['restart']),
          ' of your runtime. This may take a few minutes.  Would you like to proceed? ',
          b(['(You will not lose any files.)'])
        ]) : p([
          'Increasing the disk size or changing the number of workers (when the number of workers is >2) results in a real-time update to your runtime. ',
          'During this update, you can continue to work'
        ]),
```
but we should check with Isaac to make sure.

Reply:
your suggestion seems to be consistent with other snippets I found here 

```
   isFileImportCurrMode ? div([
              'Select the ',
              h(TooltipTrigger, { content: 'Tab Separated Values', side: 'bottom' },
                [span({ style: { textDecoration: 'underline dashed' } }, 'TSV')]),
              ' file containing your data: '
            ]) : div(['Copy and paste tab separated data here:']),
```
cc @zarsky-broad  ?

Reply:
When both cases are multiline, I prefer this, because I find it a little easier to see where each block starts and ends:
```js
condition ?
  div([
    ...
  ]) :
  div([
    ...
  ])
```

Reply:
And Isaac has improved our eslint rules to help encourage consistency: https://github.com/DataBiosphere/terra-ui/pull/2046